recipes/u-boot/u-boot-tools.inc

```diff
@@ -44,9 +44,7 @@ inherit auto-package-utils
 U_BOOT_TOOLS = "bmp_logo mkimage envcrc gen_eth_addr img2srec img2brec.sh \
     jtagconsole netconsole setlocalversion"
 
-GDB_TOOLS:machine = ""
```

Comment:
Huh?  There might have been missing such a line before this change, but as I am deleting the GDB_TOOLS:machine line, I don't see how/why I should by missing something now?  Could you be more specific?